README.md

```diff
@@ -1,2 +1,25 @@
 # cmssw_vscode
 Recipies to develop CMSSW using VSCode
+
+## Indexing files
+Indexing files allows VSCode to easily resolve references to imported files.
+
+### C++
+Indexing C++ files is based on the *clangd* extension alongside a *compile-commands.json* file.
+1. Install clangd.
+    1. In VSCode in the side bar go to *Extensions*.
+    2. In the search bar type **clangd**.
+    3. Click *Install*.
+    4. Click *Install in SSH: lxplus.cern.ch*.
+    5. Disable IntelliSense if you have to installed (comes with the C/C++ Microsoft extension).
+        1. In VSCode go to *File -> Settings -> Remote [SSH: lxplus.cern.ch]*.
+        2. In the search bar type **C_Cpp.intelliSenseEngine**.
+        3. Switch to *disabled*.
+2. Point *clangd* to the *compile_commands.json* file. A couple of options exist:
```

Comment:
Explain better where to find this file, people may work with different releases compiled for different architectures. I think the safest path is to look under `$CMSSW_RELEASE_BASE` variable after initalizing given CMSSW release environment


```
[grzankal@lxplus9112 src]$ source /cvmfs/cms.cern.ch/cmsset_default.sh
[grzankal@lxplus9112 src]$ cmsenv

[grzankal@lxplus9112 src]$ echo $CMSSW_RELEASE_BASE
/cvmfs/cms.cern.ch/el9_amd64_gcc12/cms/cmssw/CMSSW_14_0_6
[grzankal@lxplus9112 src]$ ls $CMSSW_RELEASE_BASE
biglib  cfipython              config  etc       include  logs  python  static
bin     compile_commands.json  doc     external  lib      objs  src     test
```

Reply:
Done.